src/plugins/kibana/public/settings/sections/indices/filebeat/index.html

```diff
@@ -0,0 +1,73 @@
+<kbn-settings-app section="indices">
+  <kbn-settings-indices>
+    <div>
+      <div class="page-header">
+        <h1>Tail a File</h1>
+        Let's send some log data to Elasticsearch.
+      </div>
+    </div>
+    <div ng-controller="kbnSettingsIndicesFilebeat">
```

Comment:
Turn this into a directive and access vars on controller instead of directly on $scope. Use controllerAs, bindToController, controller.